### ISSUE TITLE
Need for a `sessionV` relation?

### ISSUE THREAD

**Issue by RieksJ** [AmpersandTarski/ampersand#357]
As I am exercising the CRUD functionality of branch_#228 in the UETP project, I run into the situation where a user is editing a (nested) object, and as (s)he does so, ExecEngine rules want to assist this user by already inserting information that is related to the user's account. The point here is that I cannot express such rules, because even though `'_SESSION';sessionAccount` gets me the user's account, this expression will readily work in sessions where other users are logged in. 

One way to address this is to find a way to relate the (nested) object(s) that are being edited in a session to the '_SESSION'-atom of that session. I envisage relations `sessionV[SESSION*Concept]` and `sessionV[Concept*SESSION]` for arbitrary `Concept`s (like the relation `V` itself), which the ampersand-engineer may consider to be automatically populated with the pairs (`'_SESSION'`,'atom;') where 'atom' is in 'Concept' and 'atom' is being edited in `'_SESSION'`. 

Any other ideas?

**Comment by RieksJ** [comment]
In a similar way, I want to constrain the editing of certain objects to SESSIONs in which a user is logged in that somehow 'owns' the object. Currently, this should be done by creating an expression at every location in every INTERFACE where this is required. This means that we lack a single point of control, which may be a source of errors. Wouldn't it be nice if we could specify such containts as ordinary rules and be done with it?

**Comment by sjcjoosten** [comment]
Is there a difference between your proposed `sessionV[SESSION*Concept]` and the existing `'_SESSION';V[SESSION*Concept]` ?

**Comment by RieksJ** [comment]
Yes. `'_SESSION';V[SESSION*Concept]` may contain more than 1 pair, whereas `sessionV[SESSION*Concept]` contains precisely one pair (at runtime, in an INTERFACE), being (`_SESSION`,atom), where 'atom' is the atom upon which the INTERFACE acts.

I realized just now that this resembles the discussion on the harmonization of VIOLATIONs, INTERFACEs and VIEWs, where you suggested relations such as `src` and `tgt` which allow you to refer to the source or target of a specific violation.  We then also talked about relations that would allow us (within the context of some BOX) to refer to the specific atoms that are 'the source atoms' of that BOX and of parent boxes.

**Comment by sjcjoosten** [comment]
Rieks means Issue #133. I'm self-assigning since I also self-assigned Issue #133
